ansible-ipi-install/roles/node-prep/tasks/01_sub_man_register.yml

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
Not everybody is running a satellite or has permission to set a key at hosted,   do we know Verizon's story?

Think we need to make this task more optional.

Reply:
It will be. For now its simple to either comment out the task or I can write a simple flag that lets you set it. I'm not trying to write for every use case as that be impossible :)

Reply:
@sreichar - right now if activation_key or org_id is not defined, the sub manager task is skipped. Is this enough? Would you like to see an additional flag? true/false?